recipe/meta.yaml

```diff
@@ -324,7 +367,10 @@ outputs:
       track_features:
         - pytorch-cpu                                      # [cuda_compiler_version == "None"]
     requirements:
+      host:
+        - python
       run:
+        - python
```

Comment:
This isn't really necessary. I'm kinda "counting" on the collision between the different packages to only upload a single `pytorch-cpu` package

Reply:
Thanks, done.